lib/datadog/security/contrib/patcher.rb

```diff
@@ -0,0 +1,8 @@
+module Datadog
+  module Security
+    module Contrib
+      module Patcher
```

Comment:
The existing patcher is rather nice, no 🙂? https://github.com/DataDog/dd-trace-rb/blob/master/lib/ddtrace/patcher.rb

Reply:
Yes it is, that's why I took inspiration on it, and trying to highlight the common areas and differences in hope we can identify concerns to split and merge what can be merged.